index.html

```diff
@@ -1334,22 +1336,81 @@ <h2>Characters</h2>
 
-    <p>A set of rules for converting code points to or from code units is called a <a>character encoding form</a> (or just "character encoding" for short.</p>
+        
+        <table class="cpExample">
+            <tr>
+                <th style="width:10%">Emoji</th>
+                <th>Code Point Sequence</th>
+            </tr>
+            <tr>
+                <td class="bigtext">&#x1F46A;</td>
+                <td><code class="uname">U+1F46A FAMILY</code></td>
+            </tr>
+            <tr>
+                <td class="bigtext">&#x1F468;&#x200d;&#x1f469;&#x200d;&#x1f466;</td>
+                <td><code class="uname">U+1F468 U+200D U+1F469 U+200D U+1F466</code></td>
+            </tr>
+             <tr>
+                <td class="bigtext">&#x1F468;&#x200d;&#x1f469;&#x200d;&#x1f467;&#x200d;&#x1f466;</td>
+                <td><code class="uname">U+1F468 U+200D U+1F469 U+200D U+1F467 U+200D U+1F466</code></td>
+            </tr>  
+            <tr>
+                <td class="bigtext">&#x1F468;&#x1f3ff;&#x200d;&#x1f469;&#x1f3fe;&#x200d;&#x1f467;&#x1f3fc;&#x200d;&#x1f466;</td>
+                <td><code class="uname">U+1F468 U+1F3FF U+200D U+1F469 U+1F3FE U+200D U+1F467 U+1F3FC U+200D U+1F466</code></td>
+            </tr>
+        </table>
+        
+        
+        <p>Many common emoji can <em>only</em> be formed using sequences of code points, but should be treated as a single [=visual text unit=] when displaying or processing the text. You wouldn't want to put a line-break in the middle of the family!</p>
+    </aside>
+    
+    <p>Unicode [=code points=] are not the values actually used when processing or serializing text in a computing system. Text processing instead depends on sequences of fixed-size integer units called [=code units=]. There are different sizes of [=code unit=]. The most familiar of these is called a <dfn class="lint-ignore">byte</dfn> or <em>octet</em>, consisting of 8 bits.</p>
+    
+    <p>There are different sized [=code units=] used by different runtime environments. Other common sizes include 16- or 32-bit units. On the Web, the most familiar of these are the 16-bit units used by the [[DOM]], JavaScript, and the various string types in [[INFRA]]. These use 16-bit code units processed according to the rules of the UTF-16 [=character encoding=] of [[Unicode]]. (See [[[#char_string]]].)</p>
+    
+    <p>A [=character encoding form=] (or [=character encoding=] for short) are the rules for converting between [=code points=] in a [=character set=] and the [=code units=] used to store and process text. Non-Unicode character encoding forms are called [=legacy character encodings=]. For information on choosing a character encoding, see: [[[#char_choosing]]].</p>
```

Comment:
a character encoding form *is* the set of rules

or a character encoding form establishes the set of rules